themes/ezcontent_admin/ezcontent_admin.info.yml

```diff
@@ -56,3 +56,17 @@ libraries-override:
     css:
       theme:
         /core/themes/stable/css/views_ui/views_ui.admin.theme.css: css/dist/theme/views_ui.admin.theme.css
+
+
+regions:
```

Comment:
Why do we have to define the regions again ? Are they not inherited automatically ?